sklearn/metrics/tests/test_score_objects.py

```diff
@@ -973,3 +973,29 @@ def test_scorer_no_op_multiclass_select_proba():
         roc_auc_score, needs_proba=True, multi_class="ovo", labels=lr.classes_,
     )
     scorer(lr, X_test, y_test)
+
+
+def test_scorer_general_multiclass():
+    X, y = make_classification(
+        n_classes=3, n_informative=3, n_samples=30, random_state=0
+    )
+
+    # use `stratify` = y to ensure train and test sets capture all classes
+    X_train, X_test, y_train, y_test = train_test_split(
+        X, y, random_state=0, stratify=y
+    )
+
+    tree = DecisionTreeClassifier(random_state=0).fit(X_train, y_train)
+    lr = LogisticRegression(random_state=0).fit(X_train, y_train)
+
+    # don't use these scorers since they're not for multiclass
+    binary_only = [
```

Comment:
We can declare this on the top of the file as `BINARY_ONLY_SCORERS`

Reply:
So we should parametrize the test with the metric that we want. We could create a generate on the fly a set of the metrics to test.

Reply:
I ended up removing ```BINARY_SCORERS_ONLY``` altogether since I didn't need it after parameterizing the test